src/Recruit/SFA.DAS.Recruit/Application/ApplicationReview/Command/PatchApplicationReview/PatchApplicationReviewCommandHandler.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using MediatR;
+using Microsoft.AspNetCore.JsonPatch;
+using SFA.DAS.Recruit.InnerApi.Requests;
+using SFA.DAS.SharedOuterApi.Configuration;
+using SFA.DAS.SharedOuterApi.Extensions;
+using SFA.DAS.SharedOuterApi.Interfaces;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.Recruit.Application.ApplicationReview.Command.PatchApplicationReview
+{
+    public class PatchApplicationReviewCommandHandler(
+        IRecruitApiClient<RecruitApiConfiguration> recruitApiClient) : IRequestHandler<PatchApplicationReviewCommand>
+    {
+        public async Task Handle(PatchApplicationReviewCommand request, CancellationToken cancellationToken)
+        {
+            var jsonPatchApplicationReviewDocument = new JsonPatchDocument<InnerApi.Requests.ApplicationReview>();
+            jsonPatchApplicationReviewDocument.Replace(x => x.Status, request.Status);
+            jsonPatchApplicationReviewDocument.Replace(x => x.HasEverBeenEmployerInterviewing, request.HasEverBeenEmployerInterviewing);            
+            jsonPatchApplicationReviewDocument.Replace(x => x.EmployerFeedback, request.EmployerFeedback);
```

Comment:
i think this might also need a similar null check as to what you've done below as well?

Reply:
Sure I will include the checks for other properties as well